### PR TITLE
Couple of things I've noticed

### PR DESCRIPTION
**Pull request by skaplar**
Couple of minor things I've noticed

### CODE REVIEW

**Comment by skaplar** [comment]
I'm not sure what's failing.

**Comment by gocko** [comment]
I think that build is ok, just integration is failing because the previous build has stopped. Try to restart the previous build, and then rebuild this. I can't, do not have access to it ...

**Comment by Ducasse** [comment]
I will check it.

File: Chapters/PharoObjectModel/PharoObjectModel.pillar [hunk 1]
```diff
@@ -713,7 +713,7 @@ never be executed. If you forget to override one, and it is executed, an
 exception will be raised.
 
 Similarly, a class is considered abstract if one of its methods is abstract.
-Nothing actually prevents you from creating an instance of an abstract class;
```
Comment (Ducasse) [inline Chapters/PharoObjectModel/PharoObjectModel.pillar:716]: the ; is correct.

**Pull request merged**